README.md

```diff
@@ -71,7 +71,7 @@ You can find our documentation [here](https://skypilot.readthedocs.io/en/latest/
 - [Quickstart](https://skypilot.readthedocs.io/en/latest/getting-started/quickstart.html)
 - [CLI reference](https://skypilot.readthedocs.io/en/latest/reference/cli.html)
 
-## SkyPilot in 1 minute
+## SkyPilot in 1 Minute
```

Comment:
~Super nit, feel free to ignore: Our capitalization is inconsistent in the readme. Some headlines (`Learn more`, `Issues, feature requests, and questions`) use [AP style](https://apstylebook.com/ask_the_editor_faq)(only proper nouns and 1st letter in caps), others (`Getting Started`) use CMS style (sentence case except for prepositions). 

EDIT - saw you changed the rest of the headlines too. Thanks for making the capitalization consistent!